lib/index.ts

```diff
@@ -5,13 +5,17 @@ import { ClientOptions, EngageResult, ExportResult } from './types';
 
 export default class MixpanelClient {
   apiSecret: string;
+  account: string;
```

Comment:
Assuming that we will require the service account to be input in UI like `<serviceaccount_username>:<serviceaccount_secret>`

[ref](https://developer.mixpanel.com/reference/service-accounts)

Reply:
Yes, validation will be handled in either addax/bongo.